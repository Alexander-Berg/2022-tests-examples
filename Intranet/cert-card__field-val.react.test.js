import React from 'react';
import FieldVal from 'b:cert-card e:field-val';
import {mount} from 'enzyme';

it('should be a span', () => {
    const wrapper = mount(<FieldVal />);

    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
});
