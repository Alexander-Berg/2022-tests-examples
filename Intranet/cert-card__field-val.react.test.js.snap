// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be a span 1`] = `
<cert-card__field-val>
  <span
    className="cert-card__field-val"
    data-lego="react"
    style={Object {}}
  />
</cert-card__field-val>
`;
